src/project.test.js

```diff
@@ -2,23 +2,34 @@ const { project } = require('../');
 
 test('valid projection', () => {
   expect(
-    project({
```

Comment:
Better keep this test for a valid single-step projection and add a new test for multi-step projections.

Reply:
agree.